src/modules/users/Config.cpp

```diff
@@ -337,8 +372,16 @@ Config::setFullName( const QString& name )
         emit fullNameChanged( name );
 
         // Build login and hostname, if needed
-        QRegExp rx( "[^a-zA-Z0-9 ]", Qt::CaseInsensitive );
-        QString cleanName = CalamaresUtils::removeDiacritics( name ).toLower().replace( rx, " " ).simplified();
+        static QRegExp rx( "[^a-zA-Z0-9 ]", Qt::CaseInsensitive );
+#ifdef HAVE_ICU
+        QString cleanName = transliterate(name);
+        cleanName.replace("'", "");
```

Comment:
It's a bit odd to handle `'` differently (removed in transliteration, while it becomes a space without)